api/openapi-spec/swagger.json

```diff
@@ -49300,6 +49304,41 @@
      }
     }
    },
+   "io.k8s.kubernetes.pkg.api.v1.RookVolumeSource": {
+    "description": "RookVolumeSource represents a Rook volume that is persistend and attached to Kubernetes nodes.",
+    "required": [
+     "volumeID"
+    ],
+    "properties": {
+     "cluster": {
+      "description": "Cluster is the name of the rook cluster. Default is rook.",
+      "type": "string"
+     },
+     "fsType": {
+      "description": "Filesystem type of the volume that you want to mount. Tip: Ensure that the filesystem type is supported by the host operating system. Examples: \"ext4\", \"xfs\", \"ntfs\". Implicitly inferred to be \"ext4\" if unspecified. More info: http://kubernetes.io/docs/user-guide/volumes#rbd",
+      "type": "string"
+     },
+     "options": {
+      "description": "Extra Rook metadata if any.",
+      "type": "object",
+      "additionalProperties": {
+       "type": "string"
+      }
+     },
+     "readOnly": {
+      "description": "ReadOnly here will force the ReadOnly setting in VolumeMounts. Defaults to false.",
```

Comment:
ReadOnly vs readOnly in the description; I'd think the first instance should be lowercased like the key.